res/games/gameitem.css

```diff
@@ -0,0 +1,138 @@
+{%- macro playermacro(player,align) %}
```

Comment:
`macro playermacro` doesnt say much. Maybe if we combine "player", "table", "information", "row", we can describe what exactly it does.